### PR TITLE
Allow setting `certAge` via `HALLOW_CERT_AGE`.

### PR DESCRIPTION
**Pull request by paultag**

### CODE REVIEW

**Comment by paultag** [comment]
This name sucks, but it reflects the internal name we have. Should we change `certAge`? The environ var is not a great name.

**Comment by paultag** [comment]
maybe `certValidityDuration`?

**Comment by alex** [comment]
`certValidityDuration` sounds better!

**Comment by paultag** [comment]
@alex OK! I folded that in here to expose a sensible env flag!

File: main.go [hunk 1]
```diff
@@ -25,14 +25,17 @@ import (
...
-var defaultAllowedKeyTypes = []string{
-	ssh.KeyAlgoED25519,
-	ssh.KeyAlgoECDSA521,
-	ssh.KeyAlgoECDSA384,
-	ssh.KeyAlgoECDSA256,
-	ssh.KeyAlgoSKED25519,
-	ssh.KeyAlgoSKECDSA256,
-}
+var (
+	defaultCertAge         = 30 * time.Minute
```
Comment (alex) [inline main.go]: Name update here too

Comment (paultag) [reply main.go]: :100: catch

**Pull request merged**